src/msix/PAL/FileSystem/Win32/DirectoryObject.cpp

```diff
@@ -116,6 +116,13 @@ namespace MSIX {
     // IDirectoryObject
     ComPtr<IStream> DirectoryObject::OpenFile(const std::string& fileName, FileStream::Mode mode)
     {
+        // TODO: verify this on Win7

```

Comment:
>TODO: verify this on Win7 [](start = 11, length = 25)

TODONE?

Or pack only?

Reply:
TODONE and this is actually unpack only.

---
In reply to: [302261077](https://github.com/Microsoft/msix-packaging/pull/170#discussion_r302261077) [](ancestors = 302261077)